viewer/packages/utilities/src/events/EventTrigger.ts

```diff
@@ -2,10 +2,25 @@
  * Copyright 2021 Cognite AS
  */
 
+/**
+ * A view into an event viewer trigger
+ */
+export interface EventTriggerView<TListener extends (...args: any[]) => void> {
```

Comment:
I feel like FDM influenced name choice for that interface 😀. I think we can follow similar convention as we have with other interfaces in Reveal and rename it to old `EventTrigger` and then change the name of implementation class to something like `DefaultEventTrigger` or `InternalEventTrigger`.